doc/manual/meson.build

```diff
@@ -0,0 +1,75 @@
+if get_option('doc')
```

Comment:
Why don't you move this condition to the main `meson.build` file so it avoids entering and parsing this file in case the option is `false`?